tests/dataset/test_aoi.py

```diff
@@ -34,12 +35,15 @@ def test_multiband_input(self):
         aoi.close_raster()
         assert aoi.raster_closed
 
-    def test_multiband_input_band_selection(self):
+    @pytest.fixture(
+        params=[[2, 1], [1], [2], [3], [1, 3]]
+    )
+    def test_multiband_input_band_selection(self, request: SubRequest) -> None:
```

Comment:
Cool!
@ms888ekb are you able to use Pycharm's debugger with the "test_multiband_input_band_selection" pytest fixture you added? I can't get it to work even though it works when debugging a very similarly structured test in torchgeo: https://github.com/microsoft/torchgeo/blob/main/tests/samplers/test_single.py#L94

Also, when I simply run tests from test_aoi.py, I get a total of 30 tests when the fixture is added, whereas I get total=31 without. It seems to be skipping this specific test when adding fixtures. I played around a bit to try and figure out why torchgeo's test works, but don't understand...

Reply:
FYI, this is what I get when trying to run this specific test:
![image](https://user-images.githubusercontent.com/34774759/205702303-addafdae-5062-4256-9e7c-c84e3aec4636.png)

Reply:
This works:
```
    @pytest.fixture(
        params=[[2, 1], [1], [2], [3], [1, 3]]
    )
    def bands_request(self, request: SubRequest) -> List:
        return request.param

    def test_multiband_input_band_selection(self, bands_request):
        """Tests reading a multiband raster as input with band selection"""
        extract_archive(src="tests/data/spacenet.zip")
        data = read_csv("tests/tiling/tiling_segmentation_binary-multiband_ci.csv")
        row = data[0]
        aoi = AOI(raster=row['tif'], label=row['gpkg'], split=row['split'], raster_bands_request=bands_request)
        ...
```

Reply:
Submitted the fix